.circleci/config.yml

```diff
@@ -279,6 +279,17 @@ jobs:
       - *persist_to_workspace_step
```

Comment:
Looks like the test failures are related to Python 3.x not being available in the image. I'm not sure what to do here, since I believe I'm using the same test runner image everything else it.